src/Sarif.Viewer.VisualStudio/Models/FileDetailsModel.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Text;
+
+namespace Microsoft.Sarif.Viewer.Models
+{
+    public class FileDetailsModel
+    {
+        /// <summary>
```

Comment:
>summary [](start = 13, length = 7)

Sorry Vinay, mikefan hates doc comments because he finds them less readable. Please use regular comments. (I don't actually agree with him :-)

Reply:
heh, forgot about that. will fix.

Reply:
As typical w/someone who disagrees with my position, Larry fails to represent it in a nuanced way. :)

Doc comments are less readable than straight comments. And so, I avoid them on non-public things, like this private field. I also don't believe in useless comments (this one here is actually good). 

My rule is this: if you can't demonstrate that a doc comment is viewable outside reading the direct source file (i.e., if you can't get to it via IntelliSense) don't use it. Use a standard comment. Take all the energy you save from authoring structured XML to make the comment as good as possible.,


---
In reply to: [136137221](https://github.com/Microsoft/sarif-sdk/pull/611#discussion_r136137221) [](ancestors = 136137221)